library/core/src/ptr/const_ptr.rs

```diff
@@ -1681,7 +1681,7 @@ impl<T, const N: usize> *const [T; N] {
     }
 }
 
-// Equality for pointers
+/// Pointer equality is by address, as produced by the `std::pointer::addr` method.
```

Comment:
Nit: there is no `std::pointer::addr`, saying `ptr.addr()` is probably reasonable.

Reply:
Ah, you're right, https://doc.rust-lang.org/nightly/std/index.html?search=std%3A%3Apointer%3A%3Aaddr seemed to say this.

Reply:
Yeah, rustdoc does some things in interesting way when documenting syntax-y things like pointers and keywords.